### DISCUSSION TITLE
Is it Possible to Reverse Engineer Code Obfuscated with PyArmor Using a Free License?

### DISCUSSION THREAD

**Discussion by kiran-bal** [dashingsoft/pyarmor#1852] [Q&A]
Hello everyone,

I’ve been using PyArmor with a free license to obfuscate my Python code for an application I’m developing. 

Could anyone provide insights or share experiences regarding the following:

1. How secure is the obfuscation provided by PyArmor’s free license?
2. What are some common techniques that could be used to reverse engineer code obfuscated by PyArmor?
3. Are there any known tools or methods specifically designed to deobfuscate PyArmor-protected code?
4. What additional steps can I take to enhance the security of my obfuscated code beyond using PyArmor?

Thank you for your assistance!"

This way, you can gather detailed information about the effectiveness of PyArmor's obfuscation and the potential risks associated with relying solely on it.

**Comment by jondy** [comment]
Do not discussion how to reverse Pyarmor here

**Comment by jondy** [comment]
Please check this page for more security
https://pyarmor.readthedocs.io/en/latest/how-to/security.html

**Reply by kiran-bal** [reply]
Thank you for your response. I apologize if my question was inappropriate or if it violated the community guidelines. My intention was to seek guidance on legitimate usage and understanding of the library. I will refrain from discussing such topics here in the future. I will refer the given page
